src/lib/values.js

```diff
@@ -1,5 +0,0 @@
-// @flow
-
-export default {
-  barHeight: 48,
```

Comment:
Don't delete this file, all the other places that need the height will get it from here.